### PR TITLE
CTOR-900 : Refresh network-cisco-umbrella-snmp

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Refresh network-cisco-umbrella-snmp

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-umbrella-snmp.md [hunk 1]
```diff
@@ -1,103 +1,131 @@
...
-| Métrique                                       | Unité |
-| :--------------------------------------------- | :---- |
-| DNS connectivity status                        |       |
-| Local DNS connectivity status                  |       |
-| Umbrella dashboard (cloud) connectivity status |       |
-| AD connectors connectivity status              |       |
+| Métrique                      | Unité |
+|:------------------------------|:------|
+| DNS connectivity status       | N/A   |
+| Local DNS connectivity status | N/A   |
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-umbrella-snmp.md:66]:
```suggestion
| Local DNS connectivity status | N/A   |
| Umbrella dashboard (cloud) connectivity status | N/A |
| AD connectors connectivity status              | N/A |
```

File: pp/integrations/plugin-packs/procedures/network-cisco-umbrella-snmp.md [hunk 2]
```diff
@@ -1,97 +1,131 @@
...
-| Metric Name                                    | Unit  |
-| :--------------------------------------------- | :---- |
-| DNS connectivity status                        |       |
-| Local DNS connectivity status                  |       |
-| Umbrella dashboard (cloud) connectivity status |       |
-| AD connectors connectivity status              |       |
+| Metric Name                   | Unit |
+|:------------------------------|:-----|
+| DNS connectivity status       | N/A  |
+| Local DNS connectivity status | N/A  |
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/network-cisco-umbrella-snmp.md:65]:
```suggestion
| Local DNS connectivity status | N/A  |
| Umbrella dashboard (cloud) connectivity status | N/A  |
| AD connectors connectivity status              | N/A  |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-umbrella-snmp.md [hunk 3]
```diff
@@ -1,103 +1,131 @@
...
+
+> Pour obtenir ce nouveau format de métrique, incluez la valeur **--use-new-perfdata** dans la macro de service **EXTRAOPTIONS**.
 
 </TabItem>
 <TabItem value="Load-Average" label="Load-Average">
 
 | Métrique               | Unité |
-| :--------------------- | :---- |
+|:-----------------------|:------|
+| load.1m.average.count  | count |
```
[the remark this answers is not in the document]
Comment (lucie-dubrunfaut) [reply i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-umbrella-snmp.md:94]: https://github.com/centreon/centreon-plugins/blob/develop/src/snmp_standard/mode/loadaverage.pm#L115
https://github.com/centreon/centreon-plugins/blob/develop/tests/os/linux/snmp/load.robot

[the remark this answers is not in the document]
Comment (lucie-dubrunfaut) [reply i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-umbrella-snmp.md:94]: C'est le nombre de processus en attente dans la file en moyenne sur X minutes et sur les X coeurs

**Review by cg-tw — APPROVED** [review]

**Pull request merged**